geode-core/src/test/java/org/apache/geode/internal/cache/tx/PartitionedTXRegionStubTest.java

```diff
@@ -139,6 +171,15 @@ public void getEntryReturnsEntryGotFromRemote() throws Exception {
     verify(stub).trackBucketForTx(keyInfo);
   }
 
+  @Test
+  public void isBucketNotFoundExceptionReturnTrue() throws Exception {
+    PartitionedTXRegionStub stub = spy(new PartitionedTXRegionStub(txStateStub, partitionedRegion));
+    BucketNotFoundException ex = mock(BucketNotFoundException.class);
+    Throwable cause = mock(Throwable.class);
```

Comment:
cause is not being used, do you want to add another test case that BucketNotFoundException was wrapped as a cause?